### PR TITLE
Css variables, tabs

### PR DESCRIPTION
**Pull request by AlexanderIgorovych**

### CODE REVIEW

File: src/app/components/list/list.component.html [hunk 1]
```diff
@@ -8,11 +8,11 @@ <h2>ARIA Lists</h2>
...
     scrollable list with mouse interactions accounted for.
   </p>
-  <div multi-col v-center class="example">
+  <div multi-col v-center class="example" style="display: inherit;">
     <ea-markdown [doc]="role_list"></ea-markdown>
-    <div>
+    <div role_list_example>
       <label for="secondlist"> Longer List</label>
-      <div role="list" id="secondlist">
+      <div role="list" id="secondlist" style="max-height: 15em">
```
Comment (sebastiankr) [inline src/app/components/list/list.component.html]: ideally we do not want styles that are not defined in the themes. why do we use that here?

File: src/app/feature/button/button.component.html [hunk 2]
```diff
@@ -9,7 +9,7 @@ <h1>Button</h1>
     Eang has 4 core types of buttons. Add <code>ea-button</code> class in <code>&lt;button&gt;</code> tag to apply styles.
   </p>
 
-  <div multi-col center class="example" style="--ea-grid-col-min-width: 8em;">
+  <div multi-col center class="example" style="--ea-grid-col-min-width: calc(var(--ea-sizer) * 8em);">
```
Comment (sebastiankr) [inline src/app/feature/button/button.component.html]: that sizer should be part of the eang css, no?

File: src/app/feature/button/button.component.html [hunk 3]
```diff
@@ -23,7 +23,7 @@ <h1>Button</h1>
 
   <p>You can do the same with the <code>&lt;div&gt;</code>, <code>&lt;span&gt;</code>, <code>&lt;a&gt;</code> tags, but need also to add <code>role="ea-button"</code>.</p>
   
-  <div multi-col center class="example" style="--ea-grid-col-min-width: 4em;">
+  <div multi-col center class="example" style="--ea-grid-col-min-width: calc(var(--ea-sizer) * 8em);">
```
Comment (sebastiankr) [inline src/app/feature/button/button.component.html]: the sizer here should be part of eang, no?

File: src/app/feature/icon/icon.component.html [hunk 4]
```diff
@@ -273,4 +273,4 @@ <h2>Icon Set</h2>
       <div class="ea-card-footer">add-file</div>
     </div>
   </div>
-</div>
+</div>
```
Comment (sebastiankr) [inline src/app/feature/icon/icon.component.html]: make sure you have editor config extension installed, also use npm run format

File: src/app/feature/menu/menu.component.html [hunk 5]
```diff
@@ -14,11 +14,11 @@ <h1>Menu</h1>
   </p>
   <div multi-col v-center class="example">
     <ea-markdown [doc]="ea_menu"></ea-markdown>
-    <div class="ea-card">
-      <div class="ea-card-header">
+    <div class="ea-card card-dark">
+      <div class="ea-card-header" style="color: var(--ea-text-color-light);">
```
Comment (sebastiankr) [inline src/app/feature/menu/menu.component.html]: setting the color should not be necessary, right?

File: src/app/feature/table/table.component.html [hunk 6]
```diff
@@ -179,6 +179,6 @@ <h2 class="h2-table-caption">Text direction</h2>
       </div>
     </div>
   </div>
-  <h2 class="table-styling-header">Styling</h2>
-  <ea-markdown [doc]="table_css_variables"></ea-markdown>
+<!--   <h2 class="table-styling-header">Styling</h2>
+  <ea-markdown [doc]="table_css_variables"></ea-markdown> -->
 </div>
```
Comment (sebastiankr) [inline src/app/feature/table/table.component.html]: add newline

Comment (AlexanderIgorovych) [reply src/app/feature/table/table.component.html]: it seems everything is done, please check again, please

**Pull request merged**